app/Jobs/ImportRockTheVoteRecord.php

```diff
@@ -184,10 +192,11 @@ class ImportRockTheVoteRecord implements ShouldQueue
      *
      * @return void
      */
-    public function __construct($record)
+    public function __construct($record, $importFileId)
     {
         $this->config = ImportType::getConfig(ImportType::$rockTheVote);
-        $this->record = new RockTheVoteRecord($record, $this->config);
+        $this->record = new ParsedRockTheVoteRecord($record, $this->config);
```

Comment:
I'm finding the difference between `RockTheVoteRecord` and `ParsedRockTheVoteRecord` a little confusing at first glance. Would it make more sense to name the Eloquent model as `RockTheVoteLog`?